ImisRestApi/Escape/Payment/Controllers/PaymentController.cs

```diff
@@ -91,7 +91,7 @@ public async Task<IActionResult> Index([FromBody]IntentOfSinglePay payment)
         [Route("api/GetReconciliationData")]
         public IActionResult GetReconciliation([FromBody] GepgReconcMessage model)
         {
-            if (imisPayment.IsValidCall(model, 0))
+            if (imisPayment.IsValidCall(model, 2))
```

Comment:
Please define a constant somewhere and use it here. Having a number is difficult to understand and debug.